### PR TITLE
S204 header component

### PR DESCRIPTION
**Pull request by TianaJohnson**
Added the navbar and incorporated the new theme.

### CODE REVIEW

File: src/components/organisms/Header/Header.tsx [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
+  return css`
+    width: 100vw;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-size: cover;
+    background-color: ${background.primary};
+    margin-top: -50px;
+    .icon {
+      background-color: ${background.primary};
```
Comment (zacharyrose) [inline src/components/organisms/Header/Header.tsx:21]: What's the point of setting the background a second time for a child element? Should work the same if this is taken out

File: src/components/organisms/Header/Header.tsx [hunk 2]
```diff
@@ -32,17 +48,41 @@ export default function Header() {
...
+        <OnClickWrapper>
+          <Avatar
+            src="http://stump.zackrose.net/images/avatar_sm.png"
+            size="3x"
+          />
+        </OnClickWrapper>
+      </HeaderWrapper>
+      <TabNavWrapper>
+        <TabNavigation
+          tabs={[
```
Comment (zacharyrose) [inline src/components/organisms/Header/Header.tsx:71]: You want to take this as input as opposed to hard coding it

File: src/components/organisms/Header/Header.tsx [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
+    flex-direction: row;
+    justify-content: space-between;
+    background-size: cover;
+    background-color: ${background.primary};
+    margin-top: -50px;
+    .icon {
+      background-color: ${background.primary};
+    }
+    .image {
+      margin: 30px;
```
Comment (zacharyrose) [inline src/components/organisms/Header/Header.tsx:24]: Doesn't look like you're using this... Can you remove it?

File: src/components/molecules/index.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 export { default as ThumbIcons } from './ThumbIcons/ThumbIcons';
-
+export { default as TabNavigation } from './TabNavigation/TabNavigation';
```
Comment (zacharyrose) [inline src/components/molecules/index.ts:2]: Thanks for doing this!

File: src/components/organisms/Header/Header.tsx (hunk 1 above)
Comment (TianaJohnson) [reply src/components/organisms/Header/Header.tsx:21]: Good point. I think that was leftover Frome when I used the bubbles

File: src/components/organisms/Header/Header.tsx (hunk 2 above)
Comment (TianaJohnson) [reply src/components/organisms/Header/Header.tsx:71]: I am not quite sure what you mean for this one. Are you referring to the online styles?

**Pull request merged**